release/models/qos/openconfig-qos-elements.yang

```diff
@@ -560,6 +566,11 @@ submodule openconfig-qos-elements {
       description
         "User-defined name of the queue";
     }
+    leaf queue-id {
+      type uint8;
+      description
+        "Identifier of the queue";
```

Comment:
Here is an attempt to elaborate on the description of `queue-id` to clarify what the meaning of the queue-id is and when to use it.  

```suggestion
        "An optional identifier which may be required by some hardware to map 
        the named queue to a hardware queue";
```

Reply:
Thanks, good feedback - adopted as the above as the description.